client/js/ifml/tojson/index.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2018, the IFMLEdit.org project authors. Please see the
+// AUTHORS file for details. All rights reserved. Use of this source code is
+// governed by a MIT-style license that can be found in the LICENSE file.
+/*jslint node: true, nomen: true */
+"use strict";
+
+var elementRules = require('./elementrules'),
+    core = require('almost-core'),
+    createTransformer = require('almost').createTransformer;
+
+var transformer = createTransformer(
+        {
+            element: elementRules
+        },
+        core.merge(
```

Comment:
This is really similar to the new default `m2a` policy of `almost`

https://github.com/B3rn475/almostjs/blob/master/index.js#L23